CHANGELOG.md

```diff
@@ -4,8 +4,8 @@
 - []  ([#](https://github.com/airbnb/react-dates/pull/))
 -->
 
-## [Unreleased]
-- [fix] Fix date selection in the SDP ([#1530])
+## 20.1.1
+- [fix] add disabledPrev and disabledNext props to DayPickerSingleDateController
```

Comment:
please don't delete this entry; instead, add to the list.
```suggestion
- [new] add disabledPrev and disabledNext props to DayPickerSingleDateController ([#1615])
- [fix] Fix date selection in the SDP ([#1530])
```

Reply:
reverted